modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryLocalServiceImpl.java

```diff
@@ -1170,38 +1170,90 @@ public int getObjectEntryFolderObjectEntriesCount(
 			groupId, objectEntryFolderId);
 	}
 
+	@Override
+	public Map<Object, Long> getOneToManyAggregationCounts(
+			long groupId, long objectDefinitionId, long objectEntryId,
+			long objectRelationshipId, String aggregationTerm,
+			Predicate predicate, boolean related, String search, int start,
+			int end)
+		throws PortalException {
+
+		Table table = _objectFieldLocalService.getTable(
+			objectDefinitionId, aggregationTerm);
+
+		ObjectField objectField = _objectFieldLocalService.getObjectField(
+			objectDefinitionId, aggregationTerm);
+
+		DynamicObjectDefinitionTable dynamicObjectDefinitionTable =
+			_getDynamicObjectDefinitionTable(objectDefinitionId);
```

Comment:
do we need to sort here?

Reply:
https://github.com/liferay-bpm/liferay-portal/pull/4995/commits/6463e9700b46532d5c06cb5beba3a33bb06f9e48